examples/remote-deps/inspec.yml

```diff
@@ -0,0 +1,15 @@
+name: remote-deps
```

Comment:
I think we should add a readme for this profile. This should be targeting the built-up of a corporate compliance profile. I think we should execute some rules by default.